src/api/java/de/teamlapen/vampirism/api/entity/IVampirismEntityRegistry.java

```diff
@@ -101,15 +101,7 @@
     @Nullable
     BiteableEntry getEntry(CreatureEntity creature);
 
+    void processBloodValues(int multiplier);
```

Comment:
Nobody except Vampirism should ever use this, so no need to have this in the API right? Just cast the registry if necessary